src/components/BottomBarCourse.vue

```diff
@@ -86,22 +86,22 @@
 export default {
   props: {
     subject: String,
-    code: Number,
+    number: String,
     name: String,
-    credits: String,
-    semesters: Array,
+    credits: Number,
+    semesters: String,
     color: String,
     // requirementsMap: Map,
     id: Number,
-    instructors: Array, // array of strings
-    distributionCategories: Array, // array of strings
-    enrollmentInfo: Array,
+    instructors: String, // array of strings
```

Comment:
Change the comment to reflect it is a string now

Reply:
O das right